tests/testthat/test-pkgdown.R

```diff
@@ -15,20 +15,31 @@ get_indexed <- function(f = system.file('_pkgdown.yml', package = 'shiny')) {
 }
 
 test_that("All man pages have an entry in _pkgdown.yml", {
-  skip_on_cran()
+  # skip_on_cran()
```

Comment:
I think we can still skip on CRAN -- this will still be tested in CI, which we do all the time. Also, if we skip on CRAN and only test when using `load_all()` (though I'm not sure if that's possible), then we don't need to put `_reexports.json` in `inst/`.

Reply:
put `_reexports.json` in `./tools`?

Reply:
Sounds good to me.  I'm not sure if there's a `skip_` function that means "skip if package loaded without `load_all`", though. You might have to get creative for this. See how `pkgload::dev_meta` is implemented to see how to detect if a package was loaded with devtools/pkgload.

Reply:
👍

Reply:
Moved the test to the `./tools/documentation` folder.  This script is then called within `checkCurrentDocs.sh`. Like before, the test will fail if the pkgdown information is not correct.